### PR TITLE
Tmuxp - a python session manager built on libtmux

### PR DESCRIPTION
**Pull request by mjeveritt**

### CODE REVIEW

**Comment by mjeveritt** [comment]
Reminder to self:
- [ ] Check python versions, drop pypy
- [ ] Drop PYTHON_DEPS (inc.by distutils-r1)
Thanks floppym !

**Comment by monsieurp** [comment]
@mjeveritt hi! what's left to do? can we merge this PR as-is?

File: dev-python/kaptan/kaptan-0.5.8.ebuild [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} pypy pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="Configuration manager in your pocket"
```
Comment (jonasstein) [inline dev-python/kaptan/kaptan-0.5.8.ebuild]: "Configuration manager in your pocket" could be anything. Could you elaborate on this DESCRIPTION, please?

File: dev-python/kaptan/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+<pkgmetadata>
+	<maintainer type="person">
+		<email>m.j.everitt@iee.org</email>
+		<name>Michael Everitt</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+</pkgmetadata>
```
Comment (jonasstein) [inline dev-python/kaptan/metadata.xml]: longdescription is missing, you can add the github upstream here too.

Comment (mjeveritt) [reply dev-python/kaptan/metadata.xml]: Does this look better:
https://bpaste.net/show/2cc3dd0d0b68

File: dev-python/kaptan/kaptan-0.5.8.ebuild (hunk 1 above)
Comment (mjeveritt) [reply dev-python/kaptan/kaptan-0.5.8.ebuild]: I'll blame that on the shitty github description..... :grinning:

Perhaps something more fitting would be "Multiple config format parser" but that's verging on gibberish too .. ideas?!  :anguished:

File: dev-python/kaptan/metadata.xml (hunk 2 above)
Comment (jonasstein) [reply dev-python/kaptan/metadata.xml]: better, but I am not sure about the word "syntaxes"

Comment (mjeveritt) [reply dev-python/kaptan/metadata.xml]: It doesn't sound right, I agree .. but it checks out .. https://www.thefreedictionary.com/syntaxes , https://simple.wiktionary.org/wiki/syntax , http://www.memidex.com/syntaxes etc

Comment (mjeveritt) [reply dev-python/kaptan/metadata.xml]: Amended further, how's this:
https://bpaste.net/show/8cac80fca726

**Comment by mike-kfed** [comment]
What's the status on this one? Looks like all QA issues are fixed, I'd love to see that in the main tree as tmuxp is a useful tool and not easily installed using pip.

**Comment by monsieurp** [comment]
@mike-kfed Hi? What's up buddy? Have we met? Can you please try not to boss us around please?

Here's some basic rules to interact with strangers on the Internet, strangers you need something from:
1. Introduce yourself.
2. Say hi.
3. Be polite.
4. Be patient.

**Comment by monsieurp** [comment]
Looking forward to our next digital encounter.

**Comment by mike-kfed** [comment]
Sorry if that came across bossy, I just wanted to know what's missing and show interest as I may be able to help.

**Comment by monsieurp** [comment]
No harm done. We've been a bit overwhelmed with PRs lately but this one is in the queue, don't worry. We'll get to it eventually. Please bear with us. :clock1:

File: dev-python/kaptan/kaptan-0.5.8.ebuild [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+HOMEPAGE="https://kaptan.readthedocs.io/"
+
+SRC_URI="https://github.com/emre/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+LICENSE="BSD"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```
Comment (SoapGentoo) [inline dev-python/kaptan/kaptan-0.5.8.ebuild]: blocks are broken:
```
DESCRIPTION
HOMEPAGE
SRC_URI
<free line>
LICENSE
SLOT
KEYWORDS
```

File: dev-python/kaptan/kaptan-0.5.8.ebuild [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+
+DEPEND="${PYTHON_DEPS}
```
Comment (SoapGentoo) [inline dev-python/kaptan/kaptan-0.5.8.ebuild]: `disttutils-r1` doesnt need `${PYTHON_DEPS}` by default

File: dev-python/libtmux/libtmux-0.7.1.ebuild [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+HOMEPAGE="https://libtmux.git-pull.com/"
+
+SRC_URI="https://github.com/tony/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+LICENSE="BSD"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="${PYTHON_DEPS}"
```
Comment (SoapGentoo) [inline dev-python/libtmux/libtmux-0.7.1.ebuild]: ditto

File: dev-python/libtmux/libtmux-0.7.1.ebuild [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+
+DESCRIPTION="Python api for tmux"
+HOMEPAGE="https://libtmux.git-pull.com/"
+
+SRC_URI="https://github.com/tony/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+LICENSE="BSD"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```
Comment (SoapGentoo) [inline dev-python/libtmux/libtmux-0.7.1.ebuild]: fix blocks

File: dev-python/tmuxp/tmuxp-1.3.0.ebuild [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+HOMEPAGE="https://tmuxp.git-pull.com/"
+
+SRC_URI="https://github.com/tony/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+LICENSE="BSD"
+
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND="${PYTHON_DEPS}
```
Comment (SoapGentoo) [inline dev-python/tmuxp/tmuxp-1.3.0.ebuild]: same here

File: dev-python/kaptan/kaptan-0.5.8.ebuild [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+EAPI=6
+PYTHON_COMPAT=( python2_7 python3_{4,5,6} pypy pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="Configuration manager in your pocket"
+HOMEPAGE="https://kaptan.readthedocs.io/"
+
+SRC_URI="https://github.com/emre/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz
+	mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
```
Comment (SoapGentoo) [inline dev-python/kaptan/kaptan-0.5.8.ebuild]: just use the pypi mirror, as that is usually the properly prepared tarball and not some autogenerated git archive thingie

**Comment by mjeveritt** [comment]
Thanks for the reviews, guys. I think some of these are pending, but not pushed up here yet. Expect some update in the next couple of days.

**Comment by monsieurp** [comment]
@mjeveritt ping? :)

**Comment by monsieurp** [comment]
@mjeveritt ping? :(

**Comment by mgorny** [comment]
Closing given no activity.

**Comment by mjeveritt** [comment]
kaptan added in https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=2fb50753df60fea7b845f309ff4147912b9fd8d6 . Will likely revisit, if not already merged.

**Comment by mjeveritt** [comment]
Fixed in https://gitweb.gentoo.org/repo/gentoo.git/commit/?id=e2c50cb29e61200875643accdca7a0ddf599c8b7 thanks @chutz  :smiley: :+1: 
I can delete my local branch!

**Pull request closed without merging**